src/_langs/en/updates/2015-08-10-5-tricks-to-use-in-the-console-panel.markdown

```diff
@@ -0,0 +1,60 @@
+---
+rss: false
+layout: update
+published: true
+title: 5 tricks to use in the Console Panel
+date: 2015-08-10
+article:
+  written_on: 2015-08-10
+  updated_on: 2015-08-10
+authors:
+- umarhansa
+collection: updates
+type: tip
+category: tools
+product: chrome-devtools
+featured-image: /web/updates/images/2015-08-10-5-tricks-to-use-in-the-console-panel/five-console-tricks.gif
+source_name: DevTips
+source_url: https://umaar.com/dev-tips/29-five-console-tricks/
+teaserblocks:
+- heading: ""
+  description: ""
+  image: ""
+- heading: ""
+  description: ""
+  image: ""
+- heading: ""
+  description: ""
+  image: ""
+permalink: /updates/2015/08/10/5-tricks-to-use-in-the-console-panel.html
+---
+<img src="/web/updates/images/2015-08-10-5-tricks-to-use-in-the-console-panel/five-console-tricks.gif" alt="5 tricks to use in the Console Panel">
+
+#### Use the inspect() command to jump straight to a passed in DOM node
+
+<pre><code>inspect($('p'))</code></pre>

```

Comment:
Code needs to be marked like this:
{% highlight javascript %}
inspect($('p'))
{% endhighlight %}

Please change throughout.